runtime/common/src/weights/pallet_collator_selection.rs

```diff
@@ -42,10 +42,10 @@ impl<T: frame_system::Config> pallet_collator_selection::WeightInfo for Substrat
 		// Proof Size summary in bytes:
 		//  Measured:  `334 + b * (79 ±0)`
 		//  Estimated: `2809 + b * (2554 ±0)`
-		// Minimum execution time: 13_175_000 picoseconds.
-		Weight::from_parts(11_629_620, 2809)
-			// Standard Error: 9_686
-			.saturating_add(Weight::from_parts(3_219_955, 0).saturating_mul(b.into()))
+		// Minimum execution time: 13_026_000 picoseconds.
+		Weight::from_parts(11_528_896, 2809)
+			// Standard Error: 11_636
+			.saturating_add(Weight::from_parts(3_232_614, 0).saturating_mul(b.into()))
```

Comment:
How do you know what numbers to change all of these values to? They all seem somewhat arbitrary
.

Reply:
These numbers are automatically generated when we run benchmarks against the extrinsics we create. For example, [this benchmark](https://github.com/frequency-chain/frequency/blob/main/pallets/messages/src/benchmarking.rs) tests the performance of the messages pallet. When the benchmarks are executed, they run the specified code and provide measurements of execution time and resource usage. These metrics help us understand the computational cost of transactions and determine appropriate fees.